.github/workflows/build.yml

```diff
@@ -18,41 +18,54 @@ jobs:
   build_mac:
     name: Build macOS
     needs: lint
-    continue-on-error: true
     runs-on: [self-hosted, macOS]
+    env:
+      CONAN_PROFILE: clio_clang_14
     steps:
       - uses: actions/checkout@v3
         with:
           path: clio
 
+      - name: Install dependencies
+        run: |
+          brew install llvm@14 pkg-config ninja bison cmake ccache
+
+      - name: Create conan profile
+        run: |
+            echo "Creating $CONAN_PROFILE conan profile";
+            conan profile new $CONAN_PROFILE --detect --force
+            clang_path="$(brew --prefix llvm@14)/bin/clang"
+            clang_cxx_path="$(brew --prefix llvm@14)/bin/clang++"
+            conan profile update settings.compiler=clang $CONAN_PROFILE
+            conan profile update settings.compiler.version=14 $CONAN_PROFILE
+            conan profile update "conf.tools.build:compiler_executables={\"c\": \"$clang_path\", \"cpp\": \"$clang_cxx_path\"}" $CONAN_PROFILE
+            conan profile update env.CC="$clang_path" $CONAN_PROFILE
+            conan profile update env.CXX="$clang_cxx_path" $CONAN_PROFILE
+
+      - name: Restore conan cache
+        uses: actions/cache/restore@v3
+        with:
+          path: path/to/dependencies
```

Comment:
Looks like some sort of placeholder path?

Reply:
Fixed, thanks